Kernel/System/Console/Command/Dev/Tools/Shell.pm

```
@@ -0,0 +1,72 @@
+# --
+# Copyright (C) 2001-2015 OTRS AG, http://otrs.com/
+# --
+# This software comes with ABSOLUTELY NO WARRANTY. For details, see
+# the enclosed file COPYING for license information (AGPL). If you
+# did not receive this file, see http://www.gnu.org/licenses/agpl.txt.
+# --
+
+package Kernel::System::Console::Command::Dev::Tools::Shell;
+
+use strict;
+use warnings;
+
+use base qw(Kernel::System::Console::BaseCommand);
+
+our @ObjectDependencies = (
+    'Kernel::System::Main',
+);
+
+sub Configure {
+    my ( $Self, %Param ) = @_;
+
+    $Self->Description('An interactive REPL shell for the OTRS context.');
+
+    return;
+}
+
+sub PreRun {
+    my ( $Self, %Param ) = @_;
+
+    return if $Kernel::OM->Get('Kernel::System::Main')->Require(
+        'Devel::REPL',
```

You can add Silent => 1 here to suppress the error messages if not installed.